engine/src/test/java/io/camunda/zeebe/engine/processing/processinstance/ModifyProcessInstanceTerminationTest.java

```diff
@@ -573,6 +578,78 @@ public void shouldNotTerminateFlowScopeIfPendingActivation() {
         .contains(tuple(BpmnElementType.USER_TASK, "C", ProcessInstanceIntent.ELEMENT_ACTIVATED));
   }
 
+  @Test
+  public void shouldTerminateEventSubprocess() {
```

Comment:
🔧 I think it might be interesting to extend this test with a subprocess inside of the event sub process to verify the nesting works properly. It should work fine looking at the code. Can't hurt to verify it.

Reply:
Yes, I agree. But this is not implemented yet. When #9698 is implemented, we can extend the test case.